recipes/qt5reactor/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "qt5reactor" %}
+{% set version = "0.5" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "58d71f17188c896596e5f23f0eb884c966208a4db935e506d931797cbd1c0cbe" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script:
```

Comment:
This should be:
```yaml
  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```
or if there are multiple lines to execute:
```yaml
  script:
    - "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Reply:
I think my previous commit 4b092bc92e2711a2afce44a6bed7e6fc5495c539 implements this.

Reply:
That commit uses:
```yaml
 script:
    - script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```
instead of:
```yaml
 script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```
which causes the continuous integration checks (linked below) to fail with a fairly cryptic error.

Reply:
Got it, thanks! The fix is coming.

Reply:
It was a super-bad copy-paste error on Friday night in rush, fixed in https://github.com/conda-forge/staged-recipes/pull/7839/commits/7428c6d629c74e0f5ddcf67772595c0a7497971b. Good catch!